Userland/Libraries/LibGUI/TabWidget.cpp

```diff
@@ -27,6 +29,11 @@ TabWidget::TabWidget()
 
     REGISTER_MARGINS_PROPERTY("container_margins", container_margins, set_container_margins);
     REGISTER_BOOL_PROPERTY("uniform_tabs", uniform_tabs, set_uniform_tabs);
+    REGISTER_ENUM_PROPERTY("tab_position", this->tab_position, set_tab_position, TabPosition,
+        { TabPosition::Top, "Top" },
+        { TabPosition::Bottom, "Bottom" },
+        { TabPosition::Left, "Left" },
+        { TabPosition::Right, "Right" }, );
```

Comment:
Can you also add this property to the table in https://github.com/SerenityOS/serenity/blob/master/Base/usr/share/man/man5/GML-Widget-TabWidget.md?